tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts

```diff
@@ -15,33 +15,4 @@
 ////    new /*8*/a4.c();
 ////}
 
-var marker = 0;
-function goToMarker() {
-    marker++;
-    goTo.marker(marker.toString());
-}
-
-function verifyImport(name: string, assigningDisplay:ts.SymbolDisplayPart[], optionalParentName?: string) {
-    goToMarker();
-    var moduleNameDisplay = [{ text: name, kind: "aliasName" }];
-    if (optionalParentName) {
-        moduleNameDisplay = [{ text: optionalParentName, kind: "moduleName" }, { text: ".", kind: "punctuation" }].concat(moduleNameDisplay);
-    }
-    verify.verifyQuickInfoDisplayParts("alias", optionalParentName ? "export" : "", { start: test.markerByName(marker.toString()).position, length: name.length },
-        [{ text: "import", kind: "keyword" }, { text: " ", kind: "space" }].concat(moduleNameDisplay).concat(
-            { text: " ", kind: "space" }, { text: "=", kind: "operator" }, { text: " ", kind: "space" }).concat(assigningDisplay),
-        [], []);
-}
-
-var moduleMDisplay = [{ text: "m", kind: "moduleName" }];
-var moduleMDotM1Display = moduleMDisplay.concat({ text: ".", kind: "punctuation" }, { text: "m1", kind: "moduleName" });
-
-verifyImport("a1", moduleMDisplay);
-verifyImport("a1", moduleMDisplay);
-verifyImport("a2", moduleMDotM1Display);
-verifyImport("a2", moduleMDotM1Display);
-
-verifyImport("a3", moduleMDisplay, "m2");
-verifyImport("a3", moduleMDisplay, "m2");
-verifyImport("a4", moduleMDotM1Display, "m2");
-verifyImport("a4", moduleMDotM1Display, "m2");
\ No newline at end of file
+verify.baselineQuickInfo();

```

Comment:
This test was a bit difficult to decipher, so I swapped it for a baseline. Hope that's alright. If it helps, this is the diff for it against master.

<details>

```diff
diff --git a/tests/baselines/reference/quickInfoDisplayPartsInternalModuleAlias.baseline b/tests/baselines/reference/quickInfoDisplayPartsInternalModuleAlias.baseline
index 6c2047f..c476aba 100644
--- a/tests/baselines/reference/quickInfoDisplayPartsInternalModuleAlias.baseline
+++ b/tests/baselines/reference/quickInfoDisplayPartsInternalModuleAlias.baseline
@@ -1,434 +1,722 @@
 [
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 66
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "",
       "textSpan": {
         "start": 66,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "a1",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "a1",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   },
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 82
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "",
       "textSpan": {
         "start": 82,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "a1",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "a1",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   },
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 104
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "",
       "textSpan": {
         "start": 104,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "a2",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "a2",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "m1",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   },
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 123
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "",
       "textSpan": {
         "start": 123,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "a2",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "a2",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "m1",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   },
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 149
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "export",
       "textSpan": {
         "start": 149,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "m2",
+          "kind": "moduleName"
+        },
+        {
+          "text": ".",
+          "kind": "punctuation"
+        },
+        {
+          "text": "a3",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m2",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "a3",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   },
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 165
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "export",
       "textSpan": {
         "start": 165,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "m2",
+          "kind": "moduleName"
+        },
+        {
+          "text": ".",
+          "kind": "punctuation"
+        },
+        {
+          "text": "a3",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m2",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "a3",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   },
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 194
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "export",
       "textSpan": {
         "start": 194,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "m2",
+          "kind": "moduleName"
+        },
+        {
+          "text": ".",
+          "kind": "punctuation"
+        },
+        {
+          "text": "a4",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m2",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "a4",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "m1",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   },
   {
     "marker": {
       "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts",
       "position": 213
     },
     "quickInfo": {
       "kind": "alias",
       "kindModifiers": "export",
       "textSpan": {
         "start": 213,
         "length": 2
       },
       "displayParts": [
         {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "alias",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "namespace",
+          "kind": "keyword"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "m2",
+          "kind": "moduleName"
+        },
+        {
+          "text": ".",
+          "kind": "punctuation"
+        },
+        {
+          "text": "a4",
+          "kind": "aliasName"
+        },
+        {
+          "text": "\n",
+          "kind": "lineBreak"
+        },
+        {
           "text": "import",
           "kind": "keyword"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m2",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "a4",
           "kind": "aliasName"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "=",
           "kind": "operator"
         },
         {
           "text": " ",
           "kind": "space"
         },
         {
           "text": "m",
           "kind": "moduleName"
         },
         {
           "text": ".",
           "kind": "punctuation"
         },
         {
           "text": "m1",
           "kind": "moduleName"
         }
       ],
       "documentation": [],
       "tags": []
     }
   }
 ]
```

</details>